### PR TITLE
Resolved Merge Conflicts

### PR DESCRIPTION
**Pull request by DecDury**
I think I finally resolved those merge conflicts we were having. 

If y'all could checkout this branch and see if it builds for you that would be great.

Thank you,
Declan

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -36,6 +33,21 @@ android {
     }
     kotlinOptions {
         jvmTarget = '1.8'
+
+        /*********************************************************************************************************
+         | Duck tape level compile fix
```
Comment (DecDury) [inline app/build.gradle:38]: Supposedly this fix isn't needed in Kotlin version 1.6 i think. ( Found in one of the resources )

Comment (DecDury) [reply app/build.gradle:38]: Resources are a part of this comment in the code. Can't be seen on this small preview.

**Review by Emmanuel13g — APPROVED** [review]
Seems good!

**Review by Drew-Woodz — APPROVED** [review]
seems legit,

**Pull request merged**